code/config/lib/Loot/Config/Yaml.hs

```diff
@@ -54,6 +57,33 @@ instance
         parseSub :: Parse e (ConfigRec 'Partial us)
         parseSub = keyOrDefault (fromString $ symbolVal (Proxy :: Proxy l)) mempty configParser
 
+instance
+    forall l us is.
+        ( KnownSymbol l
+        , Monoid (ConfigRec 'Partial (SumSelection l : ToBranches us))
+        , OptionsFromJson (SumSelection l : ToBranches us)
+        , OptionsFromJson is
+        )
+    => OptionsFromJson ((l ::+ us) ': is)
+  where
+    configParser = (:&) <$> fmap ItemSumP parseSum <*> configParser
+      where
+        parseSum :: Parse e (ConfigRec 'Partial (SumSelection l : ToBranches us))
+        parseSum = keyOrDefault (fromString $ symbolVal (Proxy :: Proxy l)) mempty configParser
```

Comment:
Copy-paste worth moving to a separate function.

Reply:
Done this too